### ISSUE TITLE
Issue of missing dependencies

### ISSUE THREAD

**Issue by ShivamShokeen** [ngez/platform#2]
```
The target entry-point "@ngez/core" has missing dependencies:
 - @angular/cdk/a11y
 - @angular/cdk/portal
 - @angular/cdk/keycodes
 - @angular/cdk/overlay
```

I ran this ``` npm i @ngez/core --save  ``` command and it was giving error of missing dependencies. When I run ``` npm i  @angular/cdk/a11y --save  ```  gives an error of

`error Could not install from "@angular\cdk\a11y" as it does not contain a package.json file.`
